src/simulation/dynamics/reactionWheels/reactionWheelStateEffector.cpp

```diff
@@ -514,7 +536,14 @@ void ReactionWheelStateEffector::ConfigureRWRequests(double CurrentTime)
 			} else if(CmdIt->u_cmd < -this->ReactionWheelData[RWIter]->u_max) {
 				CmdIt->u_cmd = -this->ReactionWheelData[RWIter]->u_max;
 			}
-		}
+		} else {
+            // Warning for unlimited torque with potentially small spacecraft
+            static bool warningIssued = false;
+            if (!warningIssued && std::abs(CmdIt->u_cmd) > 10.0) {  // Arbitrary threshold for "large" torque
```

Comment:
we should not hard-code this threshold to 10.0.  Please make a private class variable that defaults to 10 Nm, document this in the RST module documentation, make setter/getter for this private class variable.